docs/source/refs.bib

```diff
@@ -1,3 +1,59 @@
+@misc{conneau2020unsupervised,
+      title={Unsupervised Cross-lingual Representation Learning for Speech Recognition}, 
+      author={Alexis Conneau and Alexei Baevski and Ronan Collobert and Abdelrahman Mohamed and Michael Auli},
+      year={2020},
+      eprint={2006.13979},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+@inproceedings{Gales2014SpeechRA,
+  title={Speech recognition and keyword spotting for low-resource languages: Babel project research at CUED},
+  author={Mark John Francis Gales and Kate Knill and Anton Ragni and Shakti Prasad Rath},
+  booktitle={SLTU},
+  year={2014}
+}
+@misc{ardila2020common,
```

Comment:
are these all being used with the current changes, or added in advance in preparation for future pretrained weights?

Reply:
It's the later case.